src/main/kotlin/traffic_simulation/Main.kt

```diff
@@ -21,7 +21,7 @@ fun main(args: Array<String>) {
 
 
 fun testScenarioWithInternList() {
-    val road: RoadNetwork = RoadNetwork(capacity = 7)
+    val road: RoadNetwork = RoadNetwork(capacity = 70)
```

Comment:
Wie du gesagt hattest sind die Kapazitäten alle um Faktor 10 erhöht worden. Finde ich gut - wegen den Rundungsfehlern bei Double-Werten.

Die Eingabe bei der Kapazität direkt hier in der Main zu erhöhen finde ich nicht ideal. Würde es vorziehen wenn die Erhöhung intern in unteren Funktionen erfolgt. Dann muss man in der REAMDE bei der Beschreibung der Anwendung nicht erläutern dass die Kapazität um Faktor 10 zu erhöhen ist.